__tests__/__unit__/utils/workspace.unit.test.ts

```diff
@@ -0,0 +1,157 @@
+import * as vscode from "vscode";
+import { closeOpenedTextFile, checkTextFileIsOpened } from "../../../src/utils/workspace";
+
+jest.mock("vscode");
+
+const activeTextEditor = jest.fn();
+const executeCommand = jest.fn();
+
+Object.defineProperty(vscode.window, "activeTextEditor", {get: activeTextEditor});
+Object.defineProperty(vscode.commands, "executeCommand", {value: executeCommand});
+
+describe("Checking Workspace file status method", () => {
```

Comment:
How about shortening the descriptions for better readability in test summary like this
```suggestion
describe("Workspace file status method", () => {
```